src/ActorFrame.cpp

```diff
@@ -413,17 +413,17 @@ void ActorFrame::PushChildTable(lua_State* L, const RString &sName)
 	{
 		if(a->GetName() == sName)
 		{
-			switch(found)
+			if (found == 0)
```

Comment:
This is good! IMO this makes way more sense to be an if statement like you have now.

Having it be a switch seems weird when we're doing arithmetic/logic with the variable.